handlers/resolution_resource.go

```diff
@@ -119,3 +119,51 @@ func HandleCreateResolution(svc dao.Service) http.Handler {
 		utils.WriteJSONWithStatus(w, req, transformers.ResolutionDaoToResponse(resolutionDao), http.StatusOK)
 	})
 }
+
+// HandleGetResolution retrieves a resolution stored against the Insolvency case
+func HandleGetResolution(svc dao.Service) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		vars := mux.Vars(req)
+		transactionID := utils.GetTransactionIDFromVars(vars)
+		if transactionID == "" {
+			log.ErrorR(req, fmt.Errorf("there is no transaction ID in the URL path"))
+			m := models.NewMessageResponse("transaction ID is not in the URL path")
+			utils.WriteJSONWithStatus(w, req, m, http.StatusBadRequest)
+			return
+		}
+
+		log.InfoR(req, fmt.Sprintf("start GET request for get resolution with transaction id: %s", transactionID))
+
+		// Check if transaction is closed
+		isTransactionClosed, err, httpStatus := service.CheckIfTransactionClosed(transactionID, req)
```

Comment:
Do we want to stop people from getting a resolution if the transaction is closed?